Assets/Scripts/Models/FurnitureJobs.cs

```diff
@@ -0,0 +1,214 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Xml;
+using MoonSharp.Interpreter;
+using UnityEngine;
+
+[MoonSharpUserData]
+public class FurnitureJobs
+{
+    private Furniture furniture;
+    private List<Job> activeJobs;
+    private List<Job> pausedJobs;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FurnitureJobs"/> class.
+    /// </summary>
+    /// <param name="furn">The current furniture.</param>
+    public FurnitureJobs(Furniture furn)
+    {
+        furniture = furn;
+        activeJobs = new List<Job>();
+        pausedJobs = new List<Job>();
+
+        SpawnSpotOffset = Vector2.zero;
+        WorkSpotOffset = Vector2.zero;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FurnitureJobs"/> class by copying some of the values from another instance.
+    /// </summary>
+    /// <param name="furn">The current furniture.</param>
+    /// <param name="other">The prototype furniture.</param>
+    public FurnitureJobs(Furniture furn, Furniture other)
+    {
+        furniture = furn;
+        activeJobs = new List<Job>();
+        pausedJobs = new List<Job>();
+
+        SpawnSpotOffset = other.Jobs.SpawnSpotOffset;
+        WorkSpotOffset = other.Jobs.WorkSpotOffset;
+    }
+
+    /// <summary>
+    /// Gets the spot offset where the Character will stand when he is using the furniture. This is relative to the bottom
+    /// left tile of the sprite. This can be outside of the actual furniture.
+    /// </summary>
+    /// <value>The spot offset where the Character will stand when he uses the furniture.</value>
+    public Vector2 WorkSpotOffset { get; private set; }
+
+    /// <summary>
+    /// Gets the spot offset where inventory is spawn when a Job is done with this machine.
+    /// </summary>
+    /// <value>The spawn spot offset.</value>
+    public Vector2 SpawnSpotOffset { get; private set; }
+
+    /// <summary>
+    /// Gets the <see cref="Job"/> with the specified index.
+    /// </summary>
+    /// <param name="i">The index.</param>
+    public Job this[int i]
+    {
+        get
+        {
+            return activeJobs[i];
+        }
+    }
+
```

Comment:
Yes we don't want to expose the collection since we need for people to use the Add and Remove methods. Adding or removing jobs directly from the collection will cause issues.